src/donejs-carousel-plugin.js

```diff
@@ -119,7 +119,7 @@ export const ViewModel = DefineMap.extend({
 	* @description passed in from the parent component 
 	*/
 	carouselOptions: {
-		type: 'observable',
+		type: 'any',
```

Comment:
what's the effect of doing this?

Reply:
let's the vm know that live binding isn't necessary for this property. Which it isn't, because it's not going to be changing after the initial setup.